unstructured/ingest/README.md

```diff
@@ -60,6 +60,10 @@ In checklist form, the above steps are summarized as:
 
 - [ ] Create a new module under [unstructured/ingest/connector/](unstructured/ingest/connector/) implementing the 3 abstract base classes, similar to [unstructured/ingest/connector/github.py](unstructured/ingest/connector/github.py).
   - [ ] The subclass of `BaseIngestDoc` overrides `process_file()` if extra processing logic is needed other than what is provided by [auto.partition()](unstructured/partition/auto.py).
+  - [ ] The subclass of `BaseConnectorConfig` implements a session handle to manage connections. Check [here](https://github.com/Unstructured-IO/unstructured/pull/1058/files#diff-dae96d30f58cffe1b348c036d006b48bdc7e2e47fbd7c8ec1c45d63face1542d) for a detailed example.
```

Comment:
Add: "If the IngestDoc relies on a connection or session that could be reused..." since this won't always be the case.
Also note that: the ConnectorConfig should inherit from `ConfigSessionHandleMixin` and implement `create_session_handle`. 
Likewise note that: the IngestDoc should inherit from `IngestDocSessionHandleMixin`